packages/Webkul/Velocity/src/Resources/views/shop/products/index.blade.php

```diff
@@ -169,6 +195,18 @@ class="category-block"
                         this.isLoading = false;
                         console.log(this.__('error.something_went_wrong'));
                     })
+                },
+
+                setSlidesPerPage: function (width) {
+                    if (width >= 1200) {
+                        this.slidesPerPage = 5;
+                    } else if (width < 1200 && width >= 626) {
+                        this.slidesPerPage = 3;
+                    } else if (width < 626 && width >= 400) {
+                        this.slidesPerPage = 2;
+                    } else {
+                        this.slidesPerPage = 1;
+                    }
                 }
```

Comment:
At that time when I am doing this, I think I was wrong. This can be handled with flexes with no wrap also. So that this logic can be avoided. If its time consuming then you can avoid it for now.

Reply:
Ignore as maybe we will remove vue carousel as that one is absolute so we can do it later.